x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/xpack/security/authc/TokenAuthIntegTests.java

```diff
@@ -104,6 +104,7 @@ public void testTokenServiceBootstrapOnNodeJoin() throws Exception {
         PlainActionFuture<UserToken> userTokenFuture = new PlainActionFuture<>();
         tokenService.decodeToken(response.getAccessToken(), userTokenFuture);
         assertNotNull(userTokenFuture.actionGet());
+        assertNotNull(response.getAuthenticationResponse());
```

Comment:
Nit: I think this assertion is better to be close to where the response object is defined, i.e. straight after `CreateTokenResponse response = restClient...`. It reads better this way since the test is close to the target. Also, in case it fails, it fails quickly before having to go through the token decrpytion which involves cluster traffic.